src/ConnectedMode/Hotspots/ServerHotspotStore.cs

```diff
@@ -23,9 +23,9 @@
 using System.ComponentModel.Composition;
 using SonarQube.Client.Models;
 
-namespace SonarLint.VisualStudio.IssueVisualization.Security.Hotspots
+namespace SonarLint.VisualStudio.ConnectedMode.Hotspots
```

Comment:
I get your point, but from my POV it would take too much effort to fix this simple mistake on a new class in a separate PR. I agree that any preexisting class should be moved in a separate PR. 

I moved this one because it needs to be accessible by Updater, however, ConnectedMode doesn't depend on IssueVis.Sec, only the other way around. And it just generally makes more sense to have Server store in ConnectedMode along with Updater and keep the Local store in IssueVis.Sec